src/Microsoft.Data/DataFrame.IO.cs

```diff
@@ -114,22 +117,27 @@ private static Type MaxKind(Type a, Type b)
         /// <param name="guessRows">number of rows used to guess types</param>
         /// <param name="addIndexColumn">add one column with the row index</param>
         /// <returns>DataFrame</returns>
-        public static DataFrame ReadStream(Func<StreamReader> createStream,
+        public static DataFrame LoadStream(Stream csvStream,
```

Comment:
I think this should be named `LoadCsv`. `LoadStream` doesn't describe this method enough.